doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -30,19 +34,22 @@ To use the Occupancy Map Updater, it is only necessary to set the appropriate pa
 YAML Configuration file (Point Cloud)
 +++++++++++++++++++++++++++++++++++++
 
-We will have to generate a YAML configuration file for configuring the 3D sensors. Please see :panda_codedir:`this example file<config/sensors_kinect_pointcloud.yaml>` for processing point clouds.
+We will have to generate a YAML configuration file for configuring the 3D sensors. Please see :panda_codedir:`this example file <config/sensors_3d.yaml>` for processing point clouds.
```

Comment:
This link is broken, as it leads to a non-existing page: https://github.com/ros-planning/panda_moveit_config/blob/rolling-devel/config/sensors_3d.yaml

Reply:
+1. Link updated.